### DISCUSSION TITLE
led gates

### DISCUSSION THREAD

**Discussion by Strangedave** [RotorHazard/RotorHazard#620] [Ideas]
Any chance we could make rotor hazard work with Wled lights.

**Comment by HazardCreative** [comment]
Already does! See here:
* https://github.com/RotorHazard/RotorHazard/blob/main/doc/Software%20Setup.md#ws2812b-led-support
* https://github.com/RotorHazard/RotorHazard/wiki/Constructing-a-wireless-LED-display-for-RotorHazard

**Comment by Strangedave** [comment]
All of our stuff is on https://github.com/Aircoookie/WLED. We could add the pi0 to a few gates, and I understand how to do that. What I was hoping for was the main rotor hazard box sending a WiFi command to the esp8266 controllers. I am not a programmer, so it’s beyond my skill set. We do have a guy in the group though. If you are telling me this already works, I think I need more detailed instructions. Dave & Candy GoddardGarry Real EstateOffice (630)310-8315Fax (630)621-3220Mobile (630)992-3283Email - ***@***.*** On Thu, 29 Jul 2021 08:13:20 -0500  ***@***.******@***.***> wrote ---- 
Already does! See here: 
 
https://github.com/RotorHazard/RotorHazard/blob/main/doc/Software%20Setup.md#ws2812b-led-support 
https://github.com/RotorHazard/RotorHazard/wiki/Constructing-a-wireless-LED-display-for-RotorHazard

**Comment by Strangedave** [comment]
So I figured part of the stuff out, Wled has html commands that will do what I'm looking for. The commands are pretty easy to handle. I found a pretty easy to understand set of instructions here. https://tynick.com/blog/01-28-2020/controlling-wled-with-raspberry-pi-using-the-wled-api/. 



This is a video of my idea. https://youtu.be/0JwRdhh5zwE



The green command is - curl "http://192.168.86.27/win&T=1&A=128&FX=0&SX=0&R=0&G=255&B=0"



The red command is - curl "http://192.168.86.27/win&T=1&A=128&FX=0&SX=0&R=255&G=0&B=0"



it would probably be cool to do yellow for the last 10 seconds too. I'm mainly trying to skip buying the pi0 and using the much cheaper D1 Mini insted. https://www.amazon.com/s?k=d1+mini&ref=nb_sb_noss_1. I'm sure I'm just scratching the surface here too. 



I gave this to a buddy that is in school for programming also, if we can figure it out we will let you know.



Thanks in advance, and great work on rotor hazard, it is a million times better than live time. 






Dave Goddard

Garry Real Estate

Office (630)310-8315

Fax (630)621-3220

Mobile (630)992-3283

Email - ***@***.***





---- On Thu, 29 Jul 2021 10:43:34 -0500 Dave Goddard ***@***.***> wrote ----


All of our stuff is on https://github.com/Aircoookie/WLED. We could add the pi0 to a few gates, and I understand how to do that. What I was hoping for was the main rotor hazard box sending a WiFi command to the esp8266 controllers. I am not a programmer, so it’s beyond my skill set. We do have a guy in the group though. If you are telling me this already works, I think I need more detailed instructions. 







Dave & Candy Goddard

Garry Real Estate

Office (630)310-8315

Fax (630)621-3220

Mobile (630)992-3283

Email - ***@***.***






---- On Thu, 29 Jul 2021 08:13:20 -0500 ***@***.******@***.***> wrote ----





Already does! See here:

https://github.com/RotorHazard/RotorHazard/blob/main/doc/Software%20Setup.md#ws2812b-led-support

https://github.com/RotorHazard/RotorHazard/wiki/Constructing-a-wireless-LED-display-for-RotorHazard

**Reply by HazardCreative** [reply]
RH won't currently use that particular protocol, but you could definitely have your programming friend add a new LED handler that can execute HTTP requests. Have them take a look at the existing LED handler libraries, then create a new one that can use your own logic instead.

**Reply by Strangedave** [reply]
I might be able to do an additional page of html for the GUI. So instead of jamming around in your code it would just be an add on. I'll mess with that later tonight, maybe there is an easy solution
